docs/upgrade.html

```diff
@@ -74,6 +74,8 @@
         will return <code>0.0</code> in such cases to minimise the probability of breaking users who read the value of every client
         metric (via a <code>MetricsReporter</code> implementation or by calling the <code>metrics()</code> method).
         <code>org.apache.kafka.common.Metric#metricValue()</code> can be used to retrieve numeric and non-numeric metric values.</li>
+    <li>The client package <code>org.apache.kafka.common.security.auth</code> has been made public and added to the javadocs.
```

Comment:
I think we should say "The package ... in the clients jar...". Otherwise, it reads a bit weird (`client` package even though `clients` is nowhere to be seen in the package name).